canonical/snapcraft.io
# Showing featured categories

## Done
Removed all snaps and use featured snaps

## How to QA
- Go to https://snapcraft-io-4469.demos.haus/beta-store
- Check that there is no featured banner
- Check that the featured category is selected by default
- Check that featured snaps have a coloured top border

## Issue / Card
Fixes:
- https://warthogs.atlassian.net/browse/WD-7666
- https://warthogs.atlassian.net/browse/WD-7667
- https://warthogs.atlassian.net/browse/WD-7668
